pyzbar/pyzbar.py

```diff
@@ -106,11 +106,21 @@ def _decode_symbols(symbols):
             for index in _RANGEFN(zbar_symbol_get_loc_size(symbol))
         )
 
+        # since 'polygon' is very misleading if one wants to detect the QR/bar
+        # code position in order no matter how they appear in the scene
+        polygon_upright = list(map(Point._make,(
```

Comment:
I find the naming more misleading now, since `polygon_upright` implies that it is invariant in regard to 90° rotations, which is *not* the case for your variant.

Also you could rename this to `polygon` and remove the definition above.

Reply:
Please check PR #78